src/features/contact/contactverzoek/contactverzoekLogboek/LogboekOverzicht.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+  <section v-if="logboekActiviteiten?.length">
+    <utrecht-heading :level="level ? level + 1 : 3">Logboek</utrecht-heading>
```

Comment:
Ik zou verwachten dat de hoogste heading in dit component gelijk is aan level, niet level + 1